tetrateci/create_istio_release.sh

```diff
@@ -5,25 +5,48 @@ set -o pipefail
 
 # HACK : the github runner runs out of space sometimes so removing the 21 GB dotnet folder
 # Temporary thing, we should be moving to a custom runner instead.
+echo "Deleting /usr/share/dotnet to reclaim space"
 [ -d "/usr/share/dotnet" ] && sudo rm -rf /usr/share/dotnet
+echo "Deletetion complete"
+
+if [[ ${BUILD} == "fips" ]]; then
+    export GOLANG_VERSION=1.15.7b5
+    echo "Fetching FIPS compliant Go"
+    url="https://go-boringcrypto.storage.googleapis.com/go1.15.7b5.linux-amd64.tar.gz"
+    wget -O go.tgz "$url"
+    echo "cb08962897e3802cda96f4ee915ed20fbde7d5d85e688759ef523d2e6ae44851 go.tgz" | sha256sum -c -
+    sudo tar -C /usr/local -xzf go.tgz
+    rm go.tgz
+    export GOROOT=/usr/local/go
+    export PATH="$GOROOT/bin:$PATH"
+    echo "FIPS compliant Go installed"
+    go version
+else
+    echo "Build docker images since it is not a fips build" 
+    echo "  - docker" >> tetrateci/manifest.yaml.in 
+fi
 
 export ISTIO_VERSION=$TAG
```

Comment:
For fips build, our tag has to be - x.y.z-tetratefips-v{n}.
I think this release is agnostic to the tag - confirming?

Reply:
It just picks up the tag you trigger the job on.